docs/book/v3/standard-filters.md

```diff
@@ -642,6 +642,37 @@ All options can be set at instantiation or by using a related method. For exampl
 methods for `target` are `getTarget()` and `setTarget()`. You can also use the `setOptions()` method
 which accepts an array of all options.
 
+## DateTimeFormatter
+
+The filter normalize a datetime string to match the filter specified format
+
+### Supported Options
+
+The following options are supported for `Laminas\Filter\DateTimeFormatter`
+
+- `format`: a valid date format to use when formatting a string
+- `timezone` : a valid timezone to be used when handling the date
+
+### Basic Usage
+
+```php
+$filter = new \Laminas\Filter\DateTimeFormatter();
+
+echo $filter->filter('2024-01-01'); // => 2024-01-01T00:00:00+00:00
+echo $filter->filter(1_359_739_801); // => 2013-02-01T17:30:01+00:00
+echo $filter->filter(new DateTimeImmutable('2024-01-01')) // => 2024-01-01T00:00:00+00:00 
+
+$filter = new \Laminas\Filter\DateTimeFormatter([
+    'format' => 'd-m-Y'
+]);
+echo $filter->filter('2024-08-16 00:00:00'); // => 16-08-2024
+
+$filter = new \Laminas\Filter\DateTimeFormatter([
+    'timezone' => 'Europe/Paris'
+]);
+echo $filter->filter('2024-01-01'); // => 2024-01-01T00:00:00+01:00
+```
```

Comment:
The code example must be split into three separate examples:

- without any options
- with format option
- with timezone option

Each example has its own heading. This improves readability.